plugins/filter_aws/aws.c

```diff
@@ -698,23 +699,88 @@ static int get_ec2_metadata(struct flb_filter_aws *ctx)
             flb_plg_error(ctx->ins, "Failed to get Hostname");
             return -1;
         }
-        ctx->new_keys++;
     }
 
+    return 0;
+}
+
+static int get_ec2_metadata_tags(struct flb_filter_aws *ctx)
+{
+    int ret;
+
+    ctx->metadata_groups[FLB_FILTER_AWS_METADATA_GROUP_TAGS].last_execution = time(NULL);
```

Comment:
nit: last_fetch or last_attempt I feel makes more sense

Reply:
OK, I renamed this field to `last_fetch_attempt`.